packages/compass-settings/src/components/sidebar.tsx

```diff
@@ -32,26 +36,19 @@ const SettingsSideNav: React.FunctionComponent<SidebarProps> = ({
 }) => {
   return (
     <nav data-testid="settings-sidebar">
-      <ul role="menu">
+      <ul role="tablist">
         {items.map((item) => (
-          <li
-            data-testid={`sidebar-${item}-item`}
-            role="option"
-            tabIndex={0}
-            aria-selected={activeItem === item}
-            className={cx(itemStyles, {
-              [itemHoverStyles]: item !== activeItem,
-              [itemActiveStyles]: item === activeItem,
-            })}
-            key={item}
-            onKeyDown={({ key }) => {
-              if (key === 'Enter') {
-                onSelectItem(item);
-              }
-            }}
-            onClick={() => onSelectItem(item)}
-          >
-            {item}
+          <li role="tab" key={item}>
+            <button
+              className={cx(itemStyles, {
+                [itemHoverStyles]: item !== activeItem,
+                [itemActiveStyles]: item === activeItem,
+              })}
+              data-testid={`sidebar-${item}-item`}
+              onClick={() => onSelectItem(item)}
+            >
+              {item}
+            </button>
           </li>
```

Comment:
A few things still need addressing here: role should be applied to the interactive element, not to the wrapper, and element with the tab role should have aria-selected and aria-controls attributes set. See [this example](https://www.w3.org/WAI/ARIA/apg/example-index/tabs/tabs-manual.html)